Dockerfile.geth

```diff
@@ -0,0 +1,71 @@
+ARG NODE_VERSION=16.14.0
+ARG ALPINE_VERSION=3.16
+ARG PYTHON_VERSION=3.10.4
+
+FROM node:${NODE_VERSION}-alpine AS node
+
+FROM python:${PYTHON_VERSION}-alpine${ALPINE_VERSION}
+
+COPY --from=node /usr/lib /usr/lib
+COPY --from=node /usr/local/share /usr/local/share
+COPY --from=node /usr/local/lib /usr/local/lib
+COPY --from=node /usr/local/include /usr/local/include
+COPY --from=node /usr/local/bin /usr/local/bin
+
+ENV GETH_DATA_DIR=/ethereum/data/
+ENV GETH_ETHEREUM_ACCOUNT=0x944466f81e70c74c0f6b736ba72c7a69e475735a
+ENV KEEP_ETHEREUM_PASSWORD=password
+ENV NETWORK=development
+
+RUN apk update 
+RUN apk add --update --no-cache bash git geth make gcc g++ && \
+  npm install --global yarn --force
+
+# There's a transitive git problem that this fixes
+RUN git config --global url."https://".insteadOf git://
+
+COPY ./solidity-contracts /solidity-contracts
+WORKDIR /solidity-contracts
+RUN git init
+RUN yarn install
+RUN yarn build
+RUN yarn link
+RUN yarn prepack
+
+COPY ./keep-core /keep-core
+WORKDIR /keep-core
+RUN git init
+
+WORKDIR /keep-core/solidity/random-beacon
+RUN yarn install
+RUN yarn link @threshold-network/solidity-contracts
+RUN yarn build
+RUN yarn link
+RUN yarn prepack
+
+WORKDIR /keep-core/solidity/ecdsa
+RUN rm -rf .openzeppelin/unknown-*.json
+RUN yarn link @threshold-network/solidity-contracts
+RUN yarn link @keep-network/random-beacon 
+RUN yarn install
+RUN yarn build
+RUN yarn link
+RUN yarn prepack
+
+COPY ./tbtc-v2 /tbtc-v2
+WORKDIR /tbtc-v2
+RUN git init
+WORKDIR /tbtc-v2/solidity
+RUN yarn install --network-timeout 1000000 && \
+  yarn build || true
```

Comment:
Why `|| true`? Should we continue if `yarn build` failed?

Reply:
Added a comment: c42e76ab408d858861b23ac415a43e237445e09c